app/src/store/voice/call/callInvite.ts

```diff
@@ -0,0 +1,276 @@
+import {
+  createEntityAdapter,
+  createSlice,
+  miniSerializeError,
+  type PayloadAction,
+  type SerializedError,
+} from '@reduxjs/toolkit';
+import { type CallInvite as TwilioCallInvite } from '@twilio/voice-react-native-sdk';
+import { match } from 'ts-pattern';
+import {
+  getCallInfo,
+  getCallInviteInfo,
+  type CallInfo,
+  type CallInviteInfo,
+} from './';
+import { type AsyncStoreSlice } from '../../app';
+import { createTypedAsyncThunk, generateThunkActionTypes } from '../../common';
+import { callInviteMap } from '../../../util/voice';
+import { settlePromise } from '../../../util/settlePromise';
+
+const sliceName = 'callInvite' as const;
```

Comment:
what is the use of `as const`?

Reply:
It's not strictly necessary, it seems like TS will by default use the narrower `'callInvite'` const type instead of `string`. But I like the definitive clarify here, we're saying that `sliceName` will _always_ have the type `'callInvite'` and cannot be interpreted as a `string` ever.